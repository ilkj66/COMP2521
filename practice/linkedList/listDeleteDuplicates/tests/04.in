4 2 6 5 4 1 7 2 3 5 4 1 2 6 4 7 2 3 4 6 8
