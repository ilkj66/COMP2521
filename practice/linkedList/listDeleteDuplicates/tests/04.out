Enter list: 
Original list: [4] -> [2] -> [6] -> [5] -> [4] -> [1] -> [7] -> [2] -> [3] -> [5] -> [4] -> [1] -> [2] -> [6] -> [4] -> [7] -> [2] -> [3] -> [4] -> [6] -> [8] -> X
After deleting duplicates: [4] -> [2] -> [6] -> [5] -> [1] -> [7] -> [3] -> [8] -> X
